src/modules/flow/converter/converter.c

```diff
@@ -834,24 +837,179 @@ drange_to_string_convert(struct sol_flow_node *node, void *data, uint16_t port,
     return r;
 }
 
+static bool
+validate_affix(const char *s)
+{
+    const char *ptr = s;
+    unsigned percent = 0;
+
+    while (*ptr) {
+        if (*ptr == '%') {
+            percent++;
+            if (percent > 2)
+                return false;
```

Comment:
this is wrong, what's bad in "%%%%" to produce "%%"? See my other comment, it's better for us to handle suffix and prefix outside of the printf call